packages/synapse-react-client/src/components/EntityFinder/details/view/DetailsViewTableRenderers.tsx

```diff
@@ -528,12 +533,16 @@ export const DetailsViewVersionRenderer = ({
             toggleSelection({
               targetId: id,
               targetVersionNumber:
-                version === NO_VERSION_NUMBER ? undefined : version,
+                version === NO_VERSION_NUMBER_OPTION_VALUE
+                  ? undefined
+                  : version,
             })
           }}
         >
           {showLatestVersion && (
-            <option value={NO_VERSION_NUMBER}>{latestVersionText}</option>
+            <option value={NO_VERSION_NUMBER_OPTION_VALUE}>
+              {latestVersionText}
+            </option>
```

Comment:
The only place we have to maintain the "NO_VERSION_NUMBER" = -1 convention, because an `<option value` cannot be null

Reply:
Understood